models/engine/db_storage.py

```diff
@@ -88,3 +88,33 @@ def close(self):
             Remove private session attribute
         '''
         self.__session.close()
+
+    def get(self, cls, id):
+        '''
+            Retrieve an obj w/class name and id
+        '''
+        result = None
```

Comment:
Variable name 'result' might not be clear what the 'get' method is getting. Maybe use a variable name like 'obj_retrieved.'